packages/console/src/App.tsx

```diff
@@ -131,17 +132,16 @@ const WrappedStoreRoutes = observer(() => {
         store.accountConfigured = status.account;
       });
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [status]);
+  }, [status, navigate]);
 
   /**
    * Starts the socket connection.
    */
+  // biome-ignore lint/correctness/useExhaustiveDependencies: mobx observers
```

Comment:
Explicitly state the disable is related to MobX observers, since HOCs are an "old" pattern for modern React